registry/src/pages/user.js

```diff
@@ -12,48 +15,98 @@ const UserPage = () => {
   const [email, setEmail] = React.useState("");
```

Comment:
I think we have most of the states listed here inside `userReducer.js` ?

Reply:
@arteevraina this page is intended for the not logged in user to see the user page of owner/manager of any package(s), (like in PyPI [userpage](https://pypi.org/user/flaskcinema/))  , so we can't use the states inside `userReducer.js` ( as they are of the logged in user). 

React UI Route for this would be: `/users/:user`

Reply:
I think `userReducer.js` is not yet there in the code. You might have to create it and list these new states inside that. I think you are referring to `authReducer.js` which actually has the states of the "logged in user"

Reply:
sure.